websockets/client.go

```diff
@@ -3,10 +3,23 @@ package websockets
 import (
 	"log"
 	"patches/protocol"
+	"time"
 
+	"github.com/gorilla/websocket"
```

Comment:
Isn't this the same library as the one below?

Reply:
Yeah, that's a copy-paste mistake.